src/lib/core/authorization/AuthorizationManager.ts

```diff
@@ -351,7 +362,10 @@ export class AuthorizationManager {
    * Process a well-formed Authorization Requirements error response from a Globus service
    * and redirect the user to the Globus Auth login page with the necessary parameters.
    */
-  handleAuthorizationRequirementsError(response: AuthorizationRequirementsError) {
+  handleAuthorizationRequirementsError(
+    response: AuthorizationRequirementsError,
+    overrides?: Overrides,
```

Comment:
I think swapping all of these to take an `options` parameter with the same named key would be much more flexible and a bit more constant.

Reply:
Yeah, was thinking the same thing. I had named it `additionalParams` in my other comments, but I intended it to convey the same idea; Use the same named key for passing additional query parameters across all of these `handle*` methods that just construct the transport.